typescript/src/start_script.ts

```diff
@@ -0,0 +1,76 @@
+#!/usr/bin/env node
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+import { execSync } from 'child_process';
+import * as fs from 'fs';
+import * as path from 'path';
+import * as rimraf from 'rimraf';
+import * as readline from 'readline';
+const argv = require('yargs').argv;
+
+const cwd = process.cwd();
+
+const GOOGLE_GAX_PROTOS_DIR = path.join(
+  cwd,
+  'node_modules',
+  'google-gax',
+  'protos'
+);
+const SRCDIR = path.join(cwd, 'build', 'src');
+const CLI = path.join(SRCDIR, 'cli.js');
+const PLUGIN = path.join(SRCDIR, 'protoc-gen-typescript_gapic');
```

Comment:
I think the `PLUGIN` is actually in the `bin` directory, not in `SRCDIR`. Let's figure out tomorrow.
The logic to copy `PLUGIN` below should not be here, since we expect that the generator is installed locally. 

There are two options:
1. The generator is installed globally with `npm install -g gapic-generator-typescript`, then `protoc-gen-typescript_gapic` is in global Node.js `bin` directory.
2. The generator is installed locally with `npm install --save-dev gapic-generator-typescript`, then `protoc-gen-typescript_gapic` is in `./node_modules/.bin`.

In both cases, I'm guessing we can take `__dirname` and guess the right location of `protoc-gen-typescript_gapic`, and add it into `PATH`. We don't need to create this file manually anyway, since `npm` will do it for us.